README.md

```diff
@@ -133,8 +134,15 @@ class SimCLR(torch.nn.Module):
         return z
 
 
-# Use a resnet backbone.
+# Use a resnet backbone from torchvision.
 backbone = torchvision.models.resnet18()
+
+# Alternatively, you can also use a backbone from timm.
+# model = timm.create_model(model_name='efficientnet_b0')
+# backbone = nn.Sequential(
+#   *list(model.children())[:-1],
+# )
```

Comment:
I would prefer not to mention timm here because the example is already quite long for a readme :)